kim_tools/ase/core.py

```
@@ -933,263 +935,919 @@ def energy_cheat(a, dimer, large_cell_len, offset, einf):
-                          Used to compute the total energy of the FCC structure
-                          for filtering.
-        ncells: A list of the number of cells corresponding to each alat.
-                        Used to compute the number of atoms for filtering by energy.
-        leds: A list of LED values corresponding to each alat.
-                     Note that leds[i] corresponds to alats[i+2]
-        min_cutoff: The minimum cutoff distance for the model-species
-                            combination. This is used to filter alats.
-        etol: A list containing the minimum and maximum energy-per-atom
-                               bounds for filtering. Default is [5e-2, 5e2].
-        led_tol: The maximum LED value for filtering. Default is 1.0.
+    n_steps = int(math.floor((a_max - a_min) / del_a + 1.0e-9))
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)